gourmet/importers/generic_recipe_parser.py

```diff
@@ -80,7 +80,7 @@ def (match_object, full_text, attribute):
     joinable_tags = ['instructions','ingredient','ingredients',None]
     change_on_join = {'ingredient':'ingredients'}
 
-    ing_matcher = re.compile("^\s*\u2022?\u2023?\u2043?\u204C?\u204D?\u2219?\u25C9?\u25D8?\u25E6?\u2619?\u2765?\u2767?\u29BE?\u29BF?\s*(%s\s+\w+.*)"%convert.NUMBER_REGEXP)
+    ing_matcher = re.compile(r"^\s*\u2022?\u2023?\u2043?\u204C?\u204D?\u2219?\u25C9?\u25D8?\u25E6?\u2619?\u2765?\u2767?\u29BE?\u29BF?\s*(%s\s+\w+.*)"%convert.NUMBER_REGEXP)
```

Comment:
This will prevent it processing all the unicode escapes, so we probably need to do something different here.

Reply:
Never mind, the `re` module handles the escapes itself, so it doesn't matter than Python's not getting them right in the string literal.